extensions/azurecore/src/account-provider/tokenCache.ts

```diff
@@ -248,6 +242,22 @@ export default class TokenCache implements adal.TokenCache {
 			});
 	}
 
+	private decryptCache(encoding: string, encryptionParams: EncryptionParams): adal.TokenResponse[] {
+		let cacheCipher = fs.readFileSync(this._cacheSerializationPath, TokenCache.FsOptions);
+		let decipher = crypto.createDecipheriv(TokenCache.CipherAlgorithm, encryptionParams.key, encryptionParams.initializationVector);
+		let cacheJson = decipher.update(cacheCipher, 'hex', <any>encoding);
+		cacheJson += decipher.final(<any>encoding);
```

Comment:
You can avoid these type casts by making `encoding` match the type that `decipher.update` is expecting: `crypto.Utf8AsciiBinaryEncoding`

Reply:
I missed that type being exported.  I'll update.